src/cloudWatchLogs/commands/searchLogGroup.ts

```diff
@@ -18,25 +18,43 @@ import { createInputBox } from '../../shared/ui/inputPrompter'
 import { createURIFromArgs } from '../cloudWatchLogsUtils'
 import { DefaultCloudWatchLogsClient } from '../../shared/clients/cloudWatchLogsClient'
 import { CloudWatchLogs } from 'aws-sdk'
+import { LogGroupNode } from '../explorer/logGroupNode'
 import { highlightDocument } from '../document/logStreamDocumentProvider'
 
-export async function searchLogGroup(registry: LogStreamRegistry): Promise<void> {
+export async function searchLogGroup(node: LogGroupNode, registry: LogStreamRegistry): Promise<void> {
     let result: telemetry.Result = 'Succeeded'
+    let logGroupInfo: CloudWatchLogsGroupInfo
+    let parameters: CloudWatchLogsParameters
+    let response: SearchLogGroupWizardResponse | undefined
```

Comment:
We're doing a lot in this function now. We should split it up to reduce the number of variables.

Reply:
I just refactored it and in order to do I merged in the `wizardSubmenu` branch.